ui/components/app/snaps/snap-ui-renderer/index.scss

```diff
@@ -1,17 +1,24 @@
 @use "design-system";
 
 .snap-ui-renderer {
+  $height-screen-sm-min: 720px;
   $width-screen-sm-min: 85vw;
   $width-screen-md-min: 80vw;
   $width-screen-lg-min: 62vw;
 
   &__container {
+    height: 100%;
+
     & > *:first-child {
       gap: 16px;
-      margin: 16px;
+      padding: 16px;
```

Comment:
There is a good reason this was changed to margin: https://github.com/MetaMask/metamask-extension/pull/29385

Reply:
dang. ok, thanks for catching this! I used this to fix a scroll issue but I think it's good now. Reverted with an update to remove a 100% height I added earlier [33e7963](https://github.com/MetaMask/metamask-extension/pull/31454/commits/33e79631f3b73be1b36f1165d0747f6d3fcb2b7e)

I also added a comment to prevent others from making the same mistake

interesting choice to set these styles on the first-child rather than targeting css classes. I'm curious as to why these were set 

cc: @FrederikBolding